src/theme/deployment/components/_display-group.scss

```diff
@@ -1,3 +1,10 @@
+$tool-one-bg: var(--tool-one-bg, var(--ion-color-secondary-400));
```

Comment:
I know this is something inherited, but it really seems off to me that there is so much plh-specific code within this very general display group component. Do you think it would be possible to refactor in some way to make it a bit cleaner?